src/components/SwapWidgetSkeleton.tsx

```diff
@@ -0,0 +1,89 @@
+import { StrictMode } from 'react'
+import styled from 'styled-components/macro'
+import { Theme, ThemeProvider } from 'theme'
+
+import Column from './Column'
+import Row from './Row'
+import ReverseButton from './Swap/ReverseButton'
+import { WidgetWrapper } from './Widget'
+
+const LoadingWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  justify-content: space-between;
+`
+const Blob = styled.div<{ height: string; width: string; radius?: number; darker?: boolean }>`
```

Comment:
`darker` should be renamed to `module` or `isModule`, as the theming is configurable.